app/main.py

```diff
@@ -16,4 +16,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = [lucky_number, pi, one_is_a_prime_number, name,
+                    my_favourite_films, profile_info, marks,
+                    collection_of_coins]
+immutable = []
+mutable = []
+for char in sorted_variables:
+    if type(char) == int or type(char) == float or type(char) == tuple\
+            or type(char) == bool or type(char) == str:
+        immutable.append(char)
+    else:
+        mutable.append(char)
+sorted_variables = {
+    "mutable": mutable,
+    "immutable": immutable
+}
+print(sorted_variables)
```

Comment:
You should just put the given variables inside the dictionary. No need to dynamically determine the type